include/seqan3/search/all.hpp

```diff
@@ -20,34 +20,35 @@
  * component for searching large amounts of data and are used for tools such as read mappers, assemblers or protein
  * search tools.
  *
- * \if DEV
- * There are currently two major kind of indices: FM indices and k-mer indices (also known as q-gram
- * indices).
+ * \if KMER
+ * SeqAn currently implements two kinds of indices: FM indices and k-mer indices (also known as q-gram indices).
+ * Generally speaking, k-mer indices support very fast searching of exact k-mers (strings of length k) or k-mers with
+ * predefined wildcard positions, which match with any character.
+ * FM indices on the other hand are more versatile and work
+ * with arbitrary pattern lengths and error numbers or positions.
+ * \else
+ * SeqAn currently implements only the FM index and a k-mer index is planned. The FM index works
+ * with arbitrary pattern lengths and error numbers or positions.
+ * \endif
  *
+ * \if DEV
  * \todo Elaborate on that (space consumption for growing k, maybe a rule of thumb).
- *
- * Generally speaking k-mer indices support very fast searching of exact k-mers (strings of length k) or k-mers with
- * predefined wildcard positions that do not have to match. FM indices on the other hand are more versatile and work
- * with arbitrary pattern lengths and error numbers / positions.
- * \todo k-mer indices are coming soon. Stay tuned!
+ *       Add a description of the DREAM index here.
```

Comment:
```suggestion
 * \todo Add a description of the DREAM index here.
```

Reply:
This is a continuation of the line above and is currently displayed in the same todo box. 
Probably you intend to make a new todo item, so I change it :)